Dockerfile

```diff
@@ -23,13 +23,17 @@ RUN R -e "install.packages('phytools')"
 RUN R -e "install.packages('rbiom')"
 RUN R -e "install.packages('remotes')"
 RUN R -e "install.packages('vegan')"
```

Comment:
can we remove vegan, rbiom etc from here?

Reply:
yup! Will do!

Reply:
i'll also add a microbiomeComputations version once the open PR there is merged